core/internal/runbranch/fork.go

```diff
@@ -0,0 +1,63 @@
+package runbranch
+
+import (
+	"context"
+
+	"github.com/Khan/genqlient/graphql"
+	"github.com/wandb/wandb/core/pkg/service"
+)
+
+type ForkBranch struct {
+	ctx         context.Context
+	client      graphql.Client
```

Comment:
client and ctx aren't used in this file, so they should be removed. If they're used in other files, consider whether they can be passed to those functions or objects more directly.

Reply:
i was debating if we should call the upsert run here, but decided against it, so i'll remove